### DISCUSSION TITLE
Display print time taken

### DISCUSSION THREAD

**Discussion by DARKFiB3R** [mriscoc/Ender3V2S1#805] [Q&A] [answered]
Is it possible to display the final print time on screen, once a print has finished?

And maybe also the actual time of day that it finished?

So something like....

------------------------------
Print Finished at 11:46pm
Total Print Time: 6h13mins
You're Awesome 😉
------------------------------

Thanks.

**Comment by mriscoc** [comment, marked as answer]
Total print time is written in the status line and in the Print Done screen. For the actual time, that can only be done by a plugin in a host because the printer doesn't have a real time clock.

**Reply by DARKFiB3R** [reply]
I think I need new glasses lol.

Thanks :)
